pkg/features/kube_features.go

```diff
@@ -774,7 +774,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	HPAContainerMetrics:                            {Default: false, PreRelease: featuregate.Alpha},
 	RootCAConfigMap:                                {Default: true, PreRelease: featuregate.Beta},
 	SizeMemoryBackedVolumes:                        {Default: false, PreRelease: featuregate.Alpha},
-	ExecProbeTimeout:                               {Default: true, PreRelease: featuregate.GA}, // lock to default in v1.21 and remove in v1.22
+	ExecProbeTimeout:                               {Default: true, PreRelease: featuregate.GA}, // remove in v1.22
```

Comment:
We have this still commented to remove in v1.22 but we should first lock it for a release before then. I think if we want to change how we're managing the release cycle for this feature we should update the KEP and bump the graduation timelines.

/hold

Reply:
Agreed we should get the KEP updated before we update this.

Reply:
https://github.com/kubernetes/enhancements/pull/2294

Reply:
@carlory can you please update the comment so it will not say "1.22". 

```suggestion
	ExecProbeTimeout:                               {Default: true, PreRelease: featuregate.GA}, // lock to default and remove after v1.22 based on KEP update
```